internal/function/commit_stats.go

```diff
@@ -0,0 +1,144 @@
+package function
+
+import (
+	"fmt"
+
+	"github.com/src-d/gitbase"
+	"github.com/src-d/gitbase/internal/utils"
+
+	"gopkg.in/src-d/go-git.v4/plumbing"
+	"gopkg.in/src-d/go-git.v4/plumbing/object"
+	"gopkg.in/src-d/go-mysql-server.v0/sql"
+)
+
+type CommitStats struct {
+	repo sql.Expression
```

Comment:
Expression properties of UDFs should all be public, so they can be accessed during transforms